dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/HibernateEventChangeLogStore.java

```diff
@@ -73,6 +73,12 @@ public class HibernateEventChangeLogStore {
           entry("dataElement", COLUMN_CHANGELOG_DATA_ELEMENT),
           entry("property", COLUMN_CHANGELOG_PROPERTY));
 
+  private static final Map<String, String> FILTERABLE_FIELDS =
```

Comment:
In here we could somehow put the type information, we know that "dataElement" is a UID (the key instead of a String could be a Pair<String, Class<?>>), so then we could validate invalid UID format in the `validateFilter` method.
I think it would be more consistent in behavior as we are not accepting invalid UIDs in filters in other APIs.